lib/LaTeXML/Package/import.sty.ltxml

```diff
@@ -31,8 +32,11 @@ DefPrimitive('\lx@set@path OptionalMatch:* {}', sub {
 DefPrimitive('\lx@append@path OptionalMatch:* {}', sub {
     my ($stomach, $star, $path) = @_;
     $path = ToString(Expand($path));
+    return unless $path;
     if (my @paths = @{ LookupValue('SEARCHPATHS') }) {
-      my $newpath = shift(@paths) . $path;
+      my $lead_path = shift(@paths);
+      $lead_path =~ s/\/+$//;
+      my $newpath = join($Config::Config{'path_sep'}, ($lead_path, $path));
```

Comment:
Shouldn't something in `Util::Pathname`, like `pathname_concat`, be covering this? There's already some pathname separator stuff in there (although possibly it should be based on `Config`).